pkg/driver/controller.go

```diff
@@ -250,7 +249,7 @@ func CreateLVMVolume(ctx context.Context, req *csi.CreateVolumeRequest,
 		}
 	}
 
-	nmap, err := getNodeMap(params.Scheduler, params.VolumeGroup)
+	nmap, err := getNodeMap(params.Scheduler, params.VgRegex)
```

Comment:
We should using VolumeGroup is VolRegex is empty here. what will happen if someone upgraded and having storageclass with volgroup only.

Reply:
Here we not introducing any other storage class param, but rather using the same volgroup param. See the other related comment below.